packages/styles/button.css

```diff
@@ -275,7 +298,8 @@ button.Link {
 .cauldron--theme-dark .Button--primary:is([disabled], [aria-disabled='true']),
 .cauldron--theme-dark .Button--secondary:is([disabled], [aria-disabled='true']),
 .cauldron--theme-dark .Button--error:is([disabled], [aria-disabled='true']),
-.cauldron--theme-dark .Button--tag:is([disabled], [aria-disabled='true']) {
+.cauldron--theme-dark .Button--tag:is([disabled], [aria-disabled='true']),
+.cauldron--theme-dark .Button--badge:is([disabled], [aria-disabled='true']) {
```

Comment:
Looks like we can remove this since the badge button should stay white in dark mode.

Reply:
Sorry about the earlier comment! My suggestion was based on a misunderstanding and outdated knowledge. So we don't need to remove the styles here.